src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -645,6 +647,60 @@ public boolean removeWorker(String name, String reason) throws IOException {
         && client.call(jedis -> removeWorkerAndPublish(jedis, name, workerChangeJson));
   }
 
+  @Override
+  public CasIndexResults removeWorkerIndexes(String workerName) throws IOException {
+    return client.call(jedis -> removeWorkerIndexes(jedis, workerName));
+  }
+
+  private CasIndexResults removeWorkerIndexes(JedisCluster cluster, String workerName) {
+
+    CasIndexResults results = new CasIndexResults();
+
+    cluster.getClusterNodes().values().stream()
+        .forEach(
+            pool -> {
+              try (Jedis jedisNode = pool.getResource()) {
+
+                // construct CAS query
+                int defaultScanCount = 10000;
+                String cursorSentinal = "0";
+                String casQuery = config.getCasPrefix() + ":*";
+                ScanParams params = new ScanParams();
+                params.match(casQuery);
+                params.count(defaultScanCount);
+
+                // iterate over all CAS entries via scanning
+                // and remove worker from the CAS keys.
+                String nextCursor = cursorSentinal;
+                do {
+
+                  ScanResult scanResult = jedisNode.scan(nextCursor, params);
+                  if (scanResult != null) {
+                    RemoveWorkerFromCasKeys(cluster, results, scanResult.getResult(), workerName);
+                    nextCursor = scanResult.getCursor();
+                  }
+                } while (!nextCursor.equals(cursorSentinal));
+              }
+            });
+
+    return results;
+  }
+
+  private void RemoveWorkerFromCasKeys(
```

Comment:
Method names should start with lowercase letter.

Reply:
fix